packages/office-ui-fabric-react/src/components/Persona/PersonaPresence.tsx

```diff
@@ -18,8 +18,10 @@ export class PersonaPresence extends React.Component<IPersonaProps, {}> {
 
   public render(): JSX.Element | null {
     let { presence, coinSize } = this.props;
-    let coinSizeWithPresenceIconStyle = coinSize ? { fontSize: coinSize / 6, lineHeight: coinSize / 3 + 'px' } : undefined;
-    let coinSizeWithPresenceStyle = coinSize ? { width: coinSize / 3, height: coinSize / 3 } : undefined;
+    const coinSizeFontScaleFactor = 6;
```

Comment:
Can move these two consts outside of the component class (underneath imports) to prevent the assignment from being done each time it calls render